### PR TITLE
Ak/login error

### PR DESCRIPTION
**Pull request by salci1990**

### CODE REVIEW

File: src/main/java/page_objects/PageObject.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+public class PageObject {
+
+    protected WebDriver driver;
+
+    public PageObject(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+}
```
Comment (bkita) [inline src/main/java/page_objects/PageObject.java:14]: please remove empty lines at the end of the class

File: src/main/java/page_objects/Header.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public Header(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(className = "login")
+    private WebElement loginLink;
+
+    @FindBy(partialLinkText = "Cart")
+    private WebElement shoppingCard;
+
```
Comment (bkita) [inline src/main/java/page_objects/Header.java:18]: please fix typo, pricesCart not Card

File: src/main/java/page_objects/Header.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    @FindBy(id = "button_order_cart")
+    private WebElement pricesCard;
+
+    public void clickLoginLink() {
+        loginLink.click();
+    }
+
```
Comment (bkita) [inline src/main/java/page_objects/Header.java:25]: please rename method to shoppingCart

File: src/main/java/page_objects/Header.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    public WebElement moveToShopingCard() {
+        return shoppingCard;
+    }
+
+    public void openShoppingCard() {
+        shoppingCard.click();
+    }
+
```
Comment (bkita) [inline src/main/java/page_objects/Header.java:33]: please rename method to priceCart

**Review by bkita — CHANGES_REQUESTED** [review]

File: src/main/java/page_objects/Header.java (hunk 3 above)
Comment (salci1990) [reply src/main/java/page_objects/Header.java:25]: done

File: src/main/java/page_objects/Header.java (hunk 2 above)
Comment (salci1990) [reply src/main/java/page_objects/Header.java:18]: done

File: src/main/java/page_objects/PageObject.java (hunk 1 above)
Comment (salci1990) [reply src/main/java/page_objects/PageObject.java:14]: done

File: src/main/java/page_objects/Header.java (hunk 3 above)
Comment (salci1990) [reply src/main/java/page_objects/Header.java:25]: done

File: src/main/java/page_objects/Header.java (hunk 4 above)
Comment (salci1990) [reply src/main/java/page_objects/Header.java:33]: done

**Pull request merged**